token.go

```diff
@@ -0,0 +1,54 @@
+package cloudconfig
+
+import (
+	"fmt"
+	"time"
+
+	tokenoperator "github.com/giantswarm/kubeadm-token-operator/service/operator"
+	microerror "github.com/giantswarm/microkit/error"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/watch"
+)
+
+const (
+	WatchTimeout = 90 * time.Second
+)
+
+func SearchToken(k8sClient kubernetes.Interface, clusterID string) (string, error) {
```

Comment:
This is something that both aws- and kvm-operator will need to do. I'm fine with moving that to operatorkit. I'm fully against implementing that in operators.